ares/gb/cartridge/board/mbc1.cpp

```diff
@@ -17,23 +17,23 @@ struct MBC1 : Interface {
 
   auto read(n16 address, n8 data) -> n8 override {
     if(address >= 0x0000 && address <= 0x3fff) {
-      return rom.read((n14)address);
+      if(io.mode) {
+        return rom.read(io.ram.bank << 19 | (n14)address);
+      } else {
```

Comment:
since both of these paths return, consider removing the else case for slightly cleaner code, this better matches coding style in the rest of ares

```c++
if(io.mode) return rom.read(io.ram.bank << 19 | (n14)address);
return rom.read((n14)address);
```

Reply:
Done.